worlds/witness/player_logic.py

```diff
@@ -226,6 +228,10 @@ def make_single_adjustment(self, adj_type: str, line: str) -> None:
                     if entity_hex in self.DOOR_ITEMS_BY_ID and item_name in self.DOOR_ITEMS_BY_ID[entity_hex]:
                         self.DOOR_ITEMS_BY_ID[entity_hex].remove(item_name)
 
+        if adj_type == "Forbidden Doors":
```

Comment:
Are this and the other uses of it supposed to be here?

Reply:
Uh oh stinky merge error